src/v/kafka/server/server.cc

```diff
@@ -129,14 +129,38 @@ server::server(
   , _gssapi_principal_mapper(
       config::shard_local_cfg().sasl_kerberos_principal_mapping.bind())
   , _krb_configurator(config::shard_local_cfg().sasl_kerberos_config.bind())
+  , _memory_fetch_sem(
+      static_cast<size_t>(
+        cfg->local().max_service_memory_per_core
+        * config::shard_local_cfg().kafka_memory_share_for_fetch()),
+      "kafka/server-mem-fetch")
   , _thread_worker(tw) {
+    vlog(klog.debug, "memory_fetch: {}", _memory_fetch_sem.available_units());
```

Comment:
nit: maybe make this message a bit more descriptive so that it makes sense without referring to the code?  Like "Starting kafka server with {} byte limit on fetch requests" or somesuch.